pysmt/logics.py

```diff
@@ -602,8 +602,13 @@ def __hash__(self):
 PYSMT_LOGICS = frozenset([QF_BOOL, QF_IDL, QF_LIA, QF_LRA, QF_RDL, QF_UF, QF_UFIDL,
                           QF_UFLIA, QF_UFLRA, QF_UFLIRA,
                           BOOL, LRA, LIA, UFLIRA, UFLRA,
-                          QF_BV, QF_UFBV])
+                          QF_BV, QF_UFBV,
+                          QF_ABV, QF_AUFBV, QF_AUFLIA, QF_ALIA
+                          #MG: QF_AX?
```

Comment:
@agriggio What is QF_AX? If we only have arrays with no additional theory for indexes isn't it a SAT problem?